setup.sh

```diff
@@ -317,7 +317,7 @@ install_hooks() {
 
 install_our_lovely_cli() {
   cd "$DEVTOOLS_DIR/our-lovely-cli"
-  npm install
+  make all
```

Comment:
Now that you're using `make`, a better alternative is to just do:
```
make -C "$DEVTOOLS_DIR/our-lovely-cli" all
````
Then we don't have to do the `cd`, which is always nice to avoid in shell scripts.